content/news/050/index.md

```diff
@@ -310,41 +358,20 @@ This can be useful to avoid the costs of a dedicated server, or to have a simila
 [lightyear_release]: https://github.com/cBournhonesque/lightyear/releases/tag/0.13.0
 [lightyear_examples]: https://github.com/cBournhonesque/lightyear/tree/main/examples
 
-
-## Popular Workgroup Issues in GitHub
-
-<!-- Up to 10 links to interesting issues -->
-
 ## Other News
 
-<!-- One-liners for plan items that haven't got their own sections. -->
-
-## Meeting Minutes
-
-<!-- Up to 10 most important notes + a link to the full details -->
-
-[See all meeting issues][label_meeting] including full text notes
-or [join the next meeting][join].
-
-[label_meeting]: https://github.com/rust-gamedev/wg/issues?q=label%3Ameeting
+Alice I. Cecile of the Bevy Foundation would like to collect community feedback
+on game development in Rust. Please fill out [her survey](https://forms.gle/kLzv5Ww3U8dLGUHU8)!
 
 ## Discussions
 
-<!-- Links to handpicked reddit/twitter/urlo/etc threads that provide
-useful information -->
-
-## Requests for Contribution
-
-<!-- Links to "good first issue"-labels or direct links to specific tasks -->
-
-## Jobs
-
-<!-- An optional section for new jobs related to Rust gamedev -->
-
-## Bonus
+LogLog games has published a [very well-written blog post](https://loglog.games/blog/leaving-rust-gamedev/>) about reasons people might have to leave Rust gamedev.
```

Comment:
```suggestion
LogLog games has published a [very well-written blog post](https://loglog.games/blog/leaving-rust-gamedev/>) about their reasons to leave Rust gamedev.
```

it's less strong than their "decision", but still the article is clearly written on their particular perspective.